sounds/models.py

```diff
@@ -80,6 +80,195 @@ def __unicode__(self):
         return self.name
 
 
+class BulkUploadProgress(models.Model):
```

Comment:
Why was this copied from lower in the file?  (remember that if you need a model for a foreignkey you can use a "String" instead). Don't worry about moving it back, it's done now...